transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -9,18 +9,38 @@ import {
 import { MediaManager } from "../../../lib/media-mgmt/mediaManager";
 import { DailyMediaManager } from "../../../lib/media-mgmt/dailyMediaManager";
 
-const SIGNALLING_TYPE = "signalling";
-
-enum SignallingMessage {
-  RENEGOTIATE = "renegotiate",
+class TrackStatusMessage {
+  type = "trackStatus";
+  receiver_index: number;
+  enabled: boolean;
+  constructor(receiver_index: number, enabled: boolean) {
+    this.receiver_index = receiver_index;
+    this.enabled = enabled;
+  }
 }
 
+const RENEGOTIATE_TYPE = "renegotiate" as const;
+type RenegotiateMessage = {
+  type: typeof RENEGOTIATE_TYPE;
+};
```

Comment:
also, why is this a type and type of vs following the pattern of TrackStatusMessage?

```suggestion
class RenegotiateMessage = {
  type = RENEGOTIATE_TYPE;
};
```

Reply:
Fixed.